terraform/modules/origin/security_groups.tf

```diff
@@ -0,0 +1,36 @@
+resource "aws_security_group" "origin_alb" {
+  name        = "fargate_${local.mode}_origin_${var.workspace_suffix}_alb"
+  vpc_id      = var.vpc_id
+  description = "${local.mode}-origin Internet-facing ALB in ${var.workspace_suffix} cluster"
+}
+
+resource "aws_security_group_rule" "service_from_origin_alb_http" {
+  for_each                 = var.apps_security_config_list
+  description              = "${each.key} receives requests from the ${local.mode}-origin ALB over HTTP"
+  type                     = "ingress"
+  from_port                = each.value.target_port
+  to_port                  = each.value.target_port
+  protocol                 = "tcp"
+  security_group_id        = each.value.security_group_id
+  source_security_group_id = aws_security_group.origin_alb.id
+}
+
```

Comment:
🤔 can we move these rules to `govuk-publishing-platform/security_group_rules.tf`? We output the security group ID already, so I guess it would be easy to shift the rules to the central place.

Reply:
we need to ask @richardTowers , it is a copy of the public-load-balancer module?